File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Open and save Word document in ASP.NET Core | Syncfusion
+description: Open and save Word document in ASP.NET Core application using Syncfusion .NET Core Word (Essential DocIO) library without Microsoft Word or interop dependencies
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and save Word document in ASP.NET Core
+
+Syncfusion Essential DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, and edit **Word** documents programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET Core**.
```

Comment:
Check and modify the sentence as per existing feedbacks

Reply:
Modified